MapleServer2/PacketHandlers/Game/SkillHandler.cs

```diff
@@ -220,16 +221,16 @@ private static void HandleDamage(GameSession session, PacketReader packet)
             return;
         }
 
-        // TODO: Verify if its the player or an ally
-        if (skillCast.IsRecovery())
-        {
-            Status status = new(skillCast, fieldPlayer.ObjectId, fieldPlayer.ObjectId, 1);
-            StatusHandler.Handle(session, status);
-
-            // TODO: Heal based on stats
-            fieldPlayer.Heal(session, status, 50);
-            return;
-        }
+        //// TODO: Verify if its the player or an ally
+        //if (skillCast.IsRecovery())
+        //{
+        //    Status status = new(skillCast, fieldPlayer.ObjectId, fieldPlayer.ObjectId, 1);
+        //    StatusHandler.Handle(session, status);
+        //
+        //    // TODO: Heal based on stats
+        //    fieldPlayer.Heal(session, status, 50);
+        //    return;
+        //}
```

Comment:
Forgot to remove this. Nice. Healing now handled in additional effects so this shouldn't be necessary.